src/js/injection_script.js

```diff
@@ -2711,6 +2711,16 @@ domainBypass("acorta-link.com", () => {
     })
 })
 
+domainBypass("hagamix.blogspot.com", () => {
+    ensureDomLoaded(() => {
+      const gotoLinkButon = document.getElementById("btn-gotolink");
+      setTimeout(function () {
+        document.getElementById("btngetlink").click();
```

Comment:
domainBypass("hagamix.blogspot.com", () => {
    ensureDomLoaded(() => {
		document.querySelector("button#btn-gotolink").click()
	})

Reply:
> domainBypass("hagamix.blogspot.com", () => { ensureDomLoaded(() => { document.querySelector("button#btn-gotolink").click() })

Did you try it? It doesn't work for me, just add your code on "Custom Bypasses" and click on my [example link](https://hagamix.blogspot.com/2015/08/10_5.html#?o=81a2ddc9fc5ccf05a57126fb146feb60b286aa8905819594e5fa2c29f60fc9857d2fdd89bbcb43b6). Nothing happens. You also forgot "})" at the end but even if you add them nothing happens.

Reply:
I have tried them on Firefox it worked, I'll try on chrome

Reply:
Don't do it through the console. It only works through the console weirdly. But if you try to do it through "Fast Forward Custom Bypasses" it doesn't work.

Reply:
> > domainBypass("hagamix.blogspot.com", () => { ensureDomLoaded(() => { document.querySelector("button#btn-gotolink").click() })
> 
> Did you try it? It doesn't work for me, just add your code on "Custom Bypasses" (click on the FastForward Extension icon and add your code there, don't use chrome developer console) and click on my [example link](https://hagamix.blogspot.com/2015/08/10_5.html#?o=81a2ddc9fc5ccf05a57126fb146feb60b286aa8905819594e5fa2c29f60fc9857d2fdd89bbcb43b6). Nothing happens. You also forgot "})" at the end but even if you add them nothing happens.

It works for me in custom bypaas, tried in Firefox only,
About the missing}, I formatted it after pasting here so, might have removed that.

Reply:
Interesting, could you enter the code in full length here? So I will try again. Something seems to be different on mine

Reply:
you're right, the behavior seems strange, it works for me  sometimes, which I suspect after I have already loaded the page a few times, might be something to do with cache, this needs more finding

Reply:
can you check if this one works, it seems overhead but it might work
`domainBypass("hagamix.blogspot.com", () => {
    ensureDomLoaded(()=>{
        var button = document.createElement("button")
    button.id="btn-gotolink";
    var body = document.getElementsByClassName('jumbotron')[0];
    body.appendChild(button);
    awaitElement("button#btn-gotolink", btn => btn.click())
    })
})`

Reply:
Works without any problems for me on Brave Browser and Firefox. I added you as Author and committed the fix.

Reply:
this doesnt work for me, i used the example link, it does nothing for me

Reply:
> this doesnt work for me, i used the example link, it does nothing for me

You need to allow pop ups on this site. Brave Browser and Firefox are blocking the pop up by default. If u click on the site it should say somewhere above on your browser tab "Allow popups from this site"

Reply:
whoah i dont want to allow them to just open up popups....

Reply:
i can finish the website without enabling popup permissions, so this is not the fix if this requires the allow popups to be enabled

Reply:
Any programmatic click would require pop up permission,
In this case we couldn't identity the href, or url so that's the only way yet.